frontend/views/user-typeahead.html

```diff
@@ -0,0 +1,9 @@
+<a>
+  <div class="user-typeahead">
+    <div class="avatar avatar-64">
+      <img ng-src="{{match.model.avatar_url}}" width="64" />
+    </div>
+    <span bind-html-unsafe="match.label | typeaheadHighlight:query"></span>
```

Comment:
Won't this allow for html injection if the username contains html chars? Maybe we should escape the username or restrict it to `^[a-zA-Z0-9\-_]+$` in the `User` model